Library/ProrationSettings.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+
+namespace Recurly
+{
+    public class ProrationSettings
+    {
+        public enum Options
+        {
+            ProratedAmount,
+            FullAmount,
+            None
+        }
+
+        /// <summary>
+        /// Proration behavior to be applied to charges for the subscription change.
+        /// Can be set to FullAmount, ProratedAmount, or None.
```

Comment:
Technically, you don't need to document "can be set to...".

Reply:
I thought it was helpful when hovering over that property on SubscriptionChange but if its not a common pattern to do this I can remove that.

Reply:
I see it as a pattern actually to note the possible values.

<img width="751" alt="Screen Shot 2024-05-07 at 12 56 29" src="https://github.com/recurly/recurly-client-dotnet/assets/96146435/fe0ae211-ce90-4b09-866e-6adf228d5c41">
<img width="825" alt="Screen Shot 2024-05-07 at 12 56 52" src="https://github.com/recurly/recurly-client-dotnet/assets/96146435/ba4e1610-8f92-4b2a-8d46-4d8a4caa69ab">

Reply:
I think that's fine for value types, such as `NetTerms`, but we and the consumer have access to the literal enum values.

Reply:
Ah got it. Right.